### DISCUSSION TITLE
Sustainability (Net Flow) changes data in the past?

### DISCUSSION THREAD

**Discussion by svscorp** [nbrown02/FlowViz-Jira#37] [Q&A]
Question, not sure is it an issue to respond to. Using this PowerBI template for quite a while and in Sustainability, noticed the following:

In February, it was showing the "blue" net flow for the 5 weeks in the row.
![Screenshot 2024-03-04 at 2 46 58 PM](https://github.com/nbrown02/FlowViz-Jira/assets/475464/a6dda825-d750-488a-af09-68ba53d21348)

Later in February, some of the Jan's week, started to show different data and some of the "ticks" became orange. Today, see screenshot, only 2 weeks are "blue", 2 weeks are "orange", one is "0". How can this change so drastically? 
![Screenshot 2024-03-04 at 2 46 12 PM](https://github.com/nbrown02/FlowViz-Jira/assets/475464/1a7d879c-7b61-447e-bfb8-309d5e98df22)

**Comment by nbrown02** [comment]
It's hard to say without seeing the underlying data for those items, but it does seem odd, certainly a change that significant. 

I am working on a new version that should capture a more accurate "In Progress" date....but hard to say what the reason/fix here is without knowing the underlying data :(

**Reply by svscorp** [reply]
What underlying data exactly would you favor? Also, I started to notice this discrepancies lately, and didn't really save previous reports :(

**Reply by nbrown02** [reply]
For those items that were completed in those weeks where the data has changed, could you provide a screenshot of the history of those using the table view section in Power BI?
